### PR TITLE
Adds translation for zones of Vietnam

### PR DESCRIPTION
**Pull request by madsnedergaard**
## Issue

Add zone titles after #5364 

## Description

Adding titles for the new zones of Vietnam! 🥳

### CODE REVIEW

**Comment by madsnedergaard** [comment]
PS. I decided to go with the English labels and not the Vietnamese ones, let me know if we should change that :)

Screenshot 2023-05-16 at 14 54 38

File: web/public/locales/en.json [hunk 1]
```diff
@@ -1814,6 +1814,18 @@
     "VN": {
       "zoneName": "Vietnam"
     },
+    "VN-C": {
+      "countryName": "Vietnam",
+      "zoneName": "Central Vietnam"
+    },
+    "VN-N": {
+      "countryName": "Vietnam",
+      "zoneName": "Southern Vietnam"
```
Comment (VIKTORVAV99) [inline web/public/locales/en.json:1823]: Surly this should be Northern Vietnam? 😉

**Review by VIKTORVAV99 — CHANGES_REQUESTED** [review]
The north will remember this 😉

File: web/public/locales/en.json (hunk 1 above)
Comment (madsnedergaard) [reply web/public/locales/en.json]: Whoops, good catch - sorry North! 😅

**Review by VIKTORVAV99 — APPROVED** [review]
LGTM! 🎉

**Pull request merged**